test/logstream/kubelogs.go

```diff
@@ -79,7 +78,9 @@ func (k *kubelogs) startForPod(eg *errgroup.Group, pod *corev1.Pod) {
 			req := k.kc.Kube.CoreV1().Pods(psn).GetLogs(pn, options)
 			stream, err := req.Stream()
 			if err != nil {
-				return err
+				k.m.Lock()
+				defer k.m.Unlock()
```

Comment:
This isn't what you want

Reply:
meh rejiggered :(

Reply:
I have Once there so it was fine at first :D